oxidation/libparsec/crates/crypto/src/common/shamir.rs

```diff
@@ -0,0 +1,65 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use super::error::CryptoError;
+use serde::{Deserialize, Serialize};
+use serde_bytes::Bytes;
+
+#[derive(Clone, Deserialize)]
+#[serde(try_from = "&Bytes")]
+pub struct ShamirShare(sharks::Share);
+
+crate::impl_key_debug!(ShamirShare);
+
+impl PartialEq for ShamirShare {
+    fn eq(&self, other: &Self) -> bool {
+        Vec::from(&self.0) == Vec::from(&other.0)
```

Comment:
No way to prevent allocation when comparing the shares ?

Reply:
a865d44fa